From c41ed80fad86b422476b5172a38df8800afc4ad3 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 11 May 2012 23:43:41 +0300
Subject: [PATCH 167/311] conf/obj.h: move #includes to the top of file

This makes obj.h consistent with the rest of files in conf/ directory.
---
 conf/obj.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/conf/obj.h b/conf/obj.h
index 0e18d89..1ed04fc 100644
--- a/conf/obj.h
+++ b/conf/obj.h
@@ -26,6 +26,14 @@
 #  include "conf/onwire_u.h" /* c2_conf_buf */
 #endif
 
+/* XXX @todo Move definitions from cfg/cfg.h to conf/schema.ff */
+#include "cfg/cfg.h"
+
+#include "lib/chan.h"  /* c2_chan */
+#include "fid/fid.h"   /* c2_fid */
+#include "lib/tlist.h" /* c2_tl, c2_tlink */
+#include "lib/types.h"
+
 /**
  * @page conf-fspec-obj Configuration Objects
  *
@@ -111,14 +119,6 @@
  * @{
  */
 
-/* XXX @todo Move definitions from cfg/cfg.h to conf/schema.ff */
-#include "cfg/cfg.h"
-
-#include "lib/chan.h"  /* c2_chan */
-#include "fid/fid.h"   /* c2_fid */
-#include "lib/tlist.h" /* c2_tl, c2_tlink */
-#include "lib/types.h"
-
 /**
  * Status of configuration object.
  *
-- 
1.8.3.2

